sdk/search/azure-search-documents/azure/search/documents/_service/_search_service_client.py

```diff
@@ -59,6 +71,163 @@ def get_service_statistics(self, **kwargs):
         result = self._client.get_service_statistics(**kwargs)
         return result.as_dict()
 
+    ### Index Operations
+
+    @distributed_trace
+    def list_indexes(self, **kwargs):
```

Comment:
Seems we could get select/request_options
https://github.com/Azure/azure-sdk-for-python/blob/97d49cd4bf250da96bbf9d60fbc7c6b97615210c/sdk/search/azure-search-documents/azure/search/documents/_service/_generated/operations/_indexes_operations.py#L108-L109

Do we want it?

Reply:
We don't want to expose select neither request_options in the convenience layer. We may want to support if-match/if-non-match, but will be in means of match-conditions.